ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -159,7 +158,6 @@ const mapDispatchToProps = (dispatch) => {
       );
     },
     getRequestAccountTabIds: () => dispatch(getRequestAccountTabIds()),
-    getCurrentWindowTab: () => dispatch(getCurrentWindowTab()),
```

Comment:
Just curious about the purpose of this removal ~

Reply:
It was unused code, and should have been a part of the previous PRs.